Manatee.Json/Schema/SchemaValidationContext.cs

```diff
@@ -113,10 +113,41 @@ public SchemaValidationContext(SchemaValidationContext source)
 			Root = source.Root;
 			RecursiveAnchor = source.RecursiveAnchor;
 			Instance = source.Instance;
-			EvaluatedPropertyNames.UnionWith(source.EvaluatedPropertyNames);
-			LocallyEvaluatedPropertyNames.UnionWith(source.LocallyEvaluatedPropertyNames);
-			ValidatedIndices.UnionWith(source.ValidatedIndices);
-			LocallyValidatedIndices.UnionWith(source.LocallyValidatedIndices);
+
+			
+			if (_evaluatedPropertyNames is null)
+			{
+				_evaluatedPropertyNames = new HashSet<string>(source.EvaluatedPropertyNames.AsEnumerable<string>());
+			}
+			else
+			{
+				_evaluatedPropertyNames.UnionWith(source.EvaluatedPropertyNames);
```

Comment:
I'm confused.  I don't think we need this; just initialize like you had it.

The other part I had mentioned is that these are backing fields for properties (line 42) which have a lazy initialization behavior.  That lazy initialization should be removed.

Reply:
NVM... I see your comment on the internal constructor.